source-academy/frontend
# Save Folder mode to localstorage

### Description

Whether the playground is in Folder mode should now persist across page loads when in the public Source Academy playground. We disable Folder mode when the playground is forced to follow the course's default Source chapter & variant because it requires Source 2+.

Part of #2176.

### Type of change


### How to test

The Folder mode is not yet ready for production and is disabled programmatically. Enable Folder mode for testing by changing the condition to `false`:
https://github.com/source-academy/frontend/blob/b90a7a249be4969e6127e08eedc06900db2a78d5/src/pages/playground/Playground.tsx#L609-L622

Check that the Folder mode stays enabled across page loads when using the Source Academy in public mode.
